cip-XXXX-Synchronizer-Migration-to-Canton-3-3/cip-XXXX-Synchronizer-Migration-to-Canton-3-3.md

```diff
@@ -0,0 +1,99 @@
+
+  ## CIP:  CIP-00XX
+<pre>
+  Title: Synchronizer Migration with Downtime to Canton 3.3
```

Comment:
I suggest to make it migration to splice 0.4.0. In the end the version people interact with directly.

Reply:
Done